README.md

```diff
@@ -332,73 +354,73 @@ sudo ./check-production.sh
-	Two subscription keys are generated (for key rotation) and both can be used for the following step.
-	* If you did request such a subscription key before, [retrieve](https://api.portal.trustedservices.intel.com/manage-subscriptions) one of your keys, which requires to log in.
-	You have two subscription keys (for key rotation) and both can be used for the following step.
+5. To setup the PCCS in the next step, you need a subscription to the [Intel PCS](https://api.portal.trustedservices.intel.com/provisioning-certification) service.
+   1. [Subscribe](https://api.portal.trustedservices.intel.com/products#product=liv-intel-software-guard-extensions-provisioning-certification-service) to Intel PCS.
+      For key rotation purpose, two subscription keys are generated.
+   2. [Retrieve](https://api.portal.trustedservices.intel.com/manage-subscriptions) one of your keys and use it in the next step.
 
-6. Configure the PCCS service:
+6. Configure the PCCS service.
 
 	```bash
 	sudo /usr/bin/pccs-configure
 	```
 
-	An example configuration you can use:
+	An example configuration:
 
 	```
 	Checking nodejs version ...
 	nodejs is installed, continue...
 	Checking cracklib-runtime ...
 	Set HTTPS listening port [8081] (1024-65535) :
 	Set the PCCS service to accept local connections only? [Y] (Y/N) :
-	Set your Intel PCS API key (Press ENTER to skip) : <Enter your Intel PCS subscription key here>
-	You didn't set Intel PCS API key. You can set it later in config/default.json.
+	Set your Intel PCS API key (Press ENTER to skip) : b1b98494660fa0de93638c531d2c0df7
 	Choose caching fill method : [LAZY] (LAZY/OFFLINE/REQ) :
-	Set PCCS server administrator password: <PCCS-ADMIN-PASSWORD>
-	Re-enter administrator password: <PCCS-ADMIN-PASSWORD>
-	Set PCCS server user password: <PCCS-SERVER-USER-PASSWORD>
-	Re-enter user password: <PCCS-SERVER-USER-PASSWORD>
+	Set PCCS server administrator password: <pccs-admin-password>
+	Re-enter administrator password: <pccs-admin-password>
+	Set PCCS server user password: <pccs-server-user-password>
+	Re-enter user password: <pccs-server-user-password>
 	Do you want to generate insecure HTTPS key and cert for PCCS service? [Y] (Y/N) :N
 	```
 
-7. Restart the PCCS service:
+   NOTE: The resulting config file is located at `/opt/intel/sgx-dcap-pccs/config/default.json`.
+
+7. Restart the PCCS service.
 
 	```bash
 	sudo systemctl restart pccs
 	```
 
-8. Verify the PCCS service is running properly:
+8. Verify the PCCS service is running properly.
 
 	```bash
 	sudo systemctl status pccs
 	```
 
-9. Platform registration.
+9. Register the platform.
 
 	The platform registration is done with the `mpa_registration_tool` tool.
 	This service is executed on system start up, registers the platform, and gets deactivated.
 	Please check the following two logs to confirm successful registration:
 
-	1. Check service logs with following command:
+	1. Check the MPA service log.
```

Comment:
```suggestion
	1. Check MPA service log:
```
@hector-cao yes, this opens the service status, but the important piece is the log at the end of the status.